app/models/concerns/activity_message_handler.rb

```diff
@@ -119,4 +138,15 @@ def activity_message_ownner(user_name)
     user_name = 'Automation System' if !user_name && Current.executed_by.present?
     user_name
   end
+
+  def determine_sla_change_type
```

Comment:
We can move this method inside `SlaActivityMessageHandler` and reduce the lines of code for this file, that way we won't need any `rubocop:disable`

Reply:
Let me check